docs/user/getting-started.md

```diff
@@ -149,6 +148,87 @@ $ kubectl get pods -n flightctl
 [...]
 ```
 
+### flightctl in ACM
+
+Create a values.yaml file with the following content, replace
+
+ 1. flightctl.MY.DOMAIN with your base domain.
+ 2. openShiftApiUrl with your actual OpenShift API URL.
+
+Please note this values file will be simplified in the future to avoid duplication
+by making use of the global.flightctl.baseDomain value.
+
+```yaml
+global:
+  flightctl:
+    baseDomain: "flightctl.MY.DOMAIN"
+    clusterLevelSecretAccess: true
+    useRoutes: true
+    networkPolicyAllowList:
+      - openshift-console
+  storageClassName: "lvms-vg1"
+flightctl:
+  api:
+    auth:
+      enabled: true
+      openShiftApiUrl: "https://api.openshift.cluster.com:6443"
+
+keycloak:
+  enabled: false
+
+# ui configuration
+flightctlUi:
+  namespace: flightctl
+  image: quay.io/flightctl/flightctl-ocp-ui:latest
```

Comment:
Should the version of the `flightctl-ocp-ui` image match the version from the helm chart?
0.1.1 in this example. (Same applies to the values.yaml file for the standalone deployment.)

Reply:
good point. I've updated the version.